index.html

```diff
@@ -100,18 +100,19 @@ <h3 class="article__title">How can I help?</h3>
                     Learn more
                   </a>
                 </p>
-              </div>
-            </div>
+              </article>
+            </section>
           </div>
-        </div>
 
-        <div class="sidebar">
+        </div>
+          <!-- </main> -->
```

Comment:
Don't forget to remove commented out code

Reply:
I use to create comments for the parents close tags so I can't read my code easily. 
Is not that a good pratice?

Reply:
Comments in HTML tend to be confusing. I would say the purpose of the semantic tags is that they show you where your elements open and close, so you shouldn't need comments too